src/install/npm.zig

```diff
@@ -35,12 +35,7 @@ const ComptimeStringMap = @import("../comptime_string_map.zig").ComptimeStringMa
 const Npm = @This();
 
 pub const Registry = struct {
-    url: URL = URL.parse("https://registry.npmjs.org/"),
-    scopes: Map = Map{},
-
-    token: string = "",
-    auth: string = "",
-
+    pub const DefaultURL = "https://registry.npmjs.org/";
```

Comment:
for zig, thje convention is usually:
- primitive types are snake_case
- Types are PascalCase
- functions are camelCase